### PR TITLE
[SC 105352] Deprecate in favor of datadog pipelines

### PR DESCRIPTION
**Pull request by brianshef**
### This PR

- Updates this forked GitHub action to merely echo a deprecation warning to the console, to prevent breaking any GitHub Actions Workflows currently referencing this particular job (highly possible due to template repos or copy/paste repos).

### Why?

- The dependencies of the original workflow action introduced a circular dependency, causing this job to fail (see SC ticket), BUT, more importantly...
- DataDog now has a (beta) integration with GitHub Actions, at https://app.datadoghq.com/ci/pipelines, which renders the purpose of this action null and void. The DataDog integration is automatic and much more human-friendly.

### CODE REVIEW

**Comment by jeremiah-brooks-rS** [comment]
@brianshef Once merged how are you going to ensure that consumers are getting this update? Are you going to delete and recreate the tags for it? From [this github search](https://github.com/search?p=1&q=org%3ArewardStyle+github-action-datadog-reporting&type=Code) it seems a lot of people had it pinned to 1.2.0 and a few to 1.3.0.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# DEPRECATED JANUARY 2022 IN FAVOR OF https://app.datadoghq.com/ci/pipelines
```
Comment (jeremiah-brooks-rS) [inline README.md:1]: @brianshef Are there steps to setup the new datadog ci pipelines for each repo?

Comment (brianshef) [reply README.md:1]: Nope, the integration is automatic with all the repos in our org using GitHub Actions.

**Comment by brianshef** [comment]
> @brianshef Once merged how are you going to ensure that consumers are getting this update? Are you going to delete and recreate the tags for it? From [this github search](https://github.com/search?p=1&q=org%3ArewardStyle+github-action-datadog-reporting&type=Code) it seems a lot of people had it pinned to 1.2.0 and a few to 1.3.0.

Ah, good point. I think this PR becomes pretty unnecessary, then. If people need to update their workflows to get the deprecation warning, then they may as well just update their workflows to remove the job altogether. I'll close this PR and archive this repo, instead.

**Pull request closed without merging**